packages/components/data-table-manager/README.md

```diff
@@ -12,6 +12,7 @@ This component enhances the `<DataTable>` component and additionally provides a
 
 - The `disableDisplaySettings` enables / disables the layout settings panel, allowing the user to select wrapping text and density display options.
 - The `disableColumnManager` enables / disables the column manager panel, allowing the user to select which columns are visible.
+- To Detach the `DatatableManager` settings dropdown from the Datatable and position it anywhere else, you would need to import a `DataTableManagerProvider` and wrap both components with the provider.
```

Comment:
I believe we would need a dedicated section to explain this behaviour.

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/bd8447871f0d99ac366b5adc9d865055c9a4506e

Thank you